dev/spacelift/dpe-sandbox/main.tf

```diff
@@ -13,7 +13,7 @@ resource "spacelift_stack" "k8s-stack" {
 
   administrative          = false
   autodeploy              = true
-  branch                  = "main"
+  branch                  = "ibcdpe-1007-monitoring"
```

Comment:
I might have said this before but it would be nice if there was a parameter to get the branch

Reply:
Yeah I need to figure out a more sustainable way. If we have multiple people working on the repo it's going to be messy.

I think once I get this PR and the VPC PR merged into main, that using trunk based development is going to be a bit nicer. Unfortunately since I am moving faster than code can be reviewed, this was the most simple thing for the mass amount of changes being implemented.

Reply:
A question related to this: what happens to these places where `branch` is set to `ibcdpe-1007-monitoring` when this is merged? Does it just get switched back to `main` before the merge?

Reply:
When I merge everything back into main I will push out a change to point the branch to main